src/xdocs/report_issue.xml

```diff
@@ -71,10 +71,10 @@ Check documentation: https://checkstyle.org/config_whitespace.html#WhitespaceAro
 
 /var/tmp $ javac Test.java
 
-/var/tmp $ cat Test.java
-public class Test{ // Doesn't warn - incorrect
-private static final int SOMETHING = 1;
-}
+/var/tmp $ cat -n Test.java
+1  public class Test{ // Doesn't warn - incorrect
```

Comment:
we avoid `-n` on purpose as output with line numbers make it hard to quickly reuse whole input file by copy-paste.
if we compare two pains "find line with violation by counting" and "removal of such line numbers to execute on local"
first pain is less and usually we have trailing comment to ease this pain ever more.

Reply:
@romani there is another option to output file with line numbers. It will cause no problem on copy-paste
I mean 
`gawk '{print $0, "// line", FNR}' Test.java`

Which will add comment with end of line to each line in file, it will result in:
```
class Test { // line 1
  private int someField; // line 2
 // line 3
  /** javadoc */ // line 4
  public void method() { // line 5
    // comment // line 6
  } // line 7
} // line 8
```

What do you think about this way?

Reply:
Too much noise :(

Reply:
Ok, I reverted this change. Only jar file version correction remains